File: tools/bench/answers/python_sonnet/s1/15_wordlens.py
sentence = "pack my box with five dozen liquor jugs"
words = sentence.split()
longest = ""
for w in words:
    if len(w) > len(longest):
        longest = w
avg = sum(len(w) for w in words) / len(words)
print(longest)
print(int(avg))

File: tools/bench/answers/rust_sonnet/s2/15_wordlens.rs
fn main() {
    let sentence = "pack my box with five dozen liquor jugs";
    let words: Vec<&str> = sentence.split_whitespace().collect();
    let longest = words.iter().max_by_key(|w| w.len()).unwrap();
    let total_len: usize = words.iter().map(|w| w.len()).sum();
    let avg = total_len / words.len();
    println!("{}", longest);
    println!("{}", avg);
}
